apps/designer/app/designer/shared/inspector/collapsible-section.tsx

```diff
@@ -51,12 +51,19 @@ export const CollapsibleSection = ({
               px: "$3",
               color: "$hiContrast",
               cursor: "default",
+              ...(isOpenFinal && { paddingBottom: 0 }),
             }}
           >
-            {isOpenFinal ? <ChevronDownIcon /> : <ChevronRightIcon />}
             <Text size="1" css={{ fontWeight: "500", flexGrow: 1 }}>
               {label}
             </Text>
+            <Flex
+              align="center"
+              justify="center"
+              css={{ marginRight: -5, opacity: ".4" }}
```

Comment:
opacity can be a number too
```suggestion
              css={{ marginRight: -5, opacity: 0.4 }}
 ```

Reply:
btw why arbitrary margin number, can be design system scale?

Reply:
why are you not using spacing from design system for the margin?

Reply:
ping